datahub-web-react/src/app/ingest/source/builder/RecipeForm/FormField.tsx

```diff
@@ -88,22 +89,26 @@ function ListField({ field, removeMargin }: ListFieldProps) {
     );
 }
 
-function SelectField({ field }: SelectFieldProps) {
+function SelectField({ field, removeMargin }: SelectFieldProps) {
     return (
-        <Form.Item
+        <StyledFormItem
             name={field.name}
             label={field.label}
             tooltip={field.tooltip}
             style={{ flexDirection: 'row', width: '80%', display: 'flex', alignItems: 'baseline' }}
+            removeMargin={!!removeMargin}
         >
             {field.options && (
-                <StyledSelectField>
+                <StyledSelectField
+                    placeholder={field?.placeholder}
```

Comment:
same here with `?` - I don't believe it's necessary

Reply:
Removed!!